torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -702,6 +702,17 @@ def set_state_dict_type(
 
     @staticmethod
     def get_state_dict_type(module: nn.Module) -> StateDictSettings:
+        """
+        Get the state_dict_type and the corresponding configurations
```

Comment:
nit: We should use double backtick for "state_dict_type" or use "state dict type" without underscores.

Reply:
Will just leave them as non-backticked